node/tessel-export.js

```diff
@@ -557,19 +562,21 @@ Tessel.Pin.prototype.addListener = function(mode, callback) {
       throw new Error(`Interrupts are not supported on pin ${this.pin}. Pins 2, 5, 6, and 7 on either port support interrupts.`);
     }
 
-    if ((mode === 'high' || mode === 'low') && !callback.listener) {
+    // For one-time 'low' or 'high' event
+    if ((mode === 'low' || mode === 'high') && !callback.listener) {
       throw new Error('Cannot use "on" with level interrupts. You can only use "once".');
     }
 
-    if (this.interruptMode !== mode) {
-      if (this.interruptMode) {
-        throw new Error(`Cannot set pin interrupt mode to ${mode}; already listening for ${this.interruptMode}`);
+    // Can't set multiple listeners when using 'low' or 'high'
+    if (this.interruptMode) {
+      if ((mode === 'low' || mode === 'high') || (this.listeners('low')[0] || this.listeners('high')[0])) {
```

Comment:
The original code won't allow multiple listeners of any kind... the new code only prevents multiple listeners if there is already a 'high' or 'low' listener set or if trying to add a 'high' or 'low' listener when any listener already exists.

To state it another way, the new code allows for any combination of 'change', 'rise' and 'fall' listeners to be set, but prevents multiple listeners when setting 'high' or 'low' listeners.